src/server.ts

```diff
@@ -260,7 +270,25 @@ export class Server<
         recordException('[Comet] Failed to handle request.')
         recordException(error)
         span.end()
-        return new Response(null, { status: 500 })
+
+        return await this.tracer.startActiveSpan('comet error handler', async span => {
+
+          // Run global after middleware, as those mostly contain cleanups, analytics, logging, etc.
+          // That would be useful to have if an error occurs earlier on.
+          // This relies on the internal deduplication of called middleware, so if the error is thrown in a global-after
+          // Middleware, it will not be called again.
+          await this.handleMiddleware({
+            middleware: this.options.after,
+            called: calledMiddleware,
+            input,
+            type: 'global-after'
+          })
```

Comment:
Is this left over from the previous pr?

Reply:
@DaniFoldi

Reply:
Yeah, this was something I was thinking about, unfortunately without the mw dependency tracking I don't think it can be implemented as we could break the dependency chain by calling global afters. What do you think?